app/controllers/v0/education_enrollment_status_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module V0
+  class EducationEnrollmentStatusController < ApplicationController
+    # GET /edcuation_enrollment_status
+    def show
+      # TODO - attach to serializer
+      render json: {}
```

Comment:
obviously placeholder until the rest of the endpoint is complete but you can do `head :ok` for a blank 200 response.